### ISSUE TITLE
New permission required

### ISSUE THREAD

**Issue by zoltanf** [recruit-mp/android-HeaderFooterGridView#3]
First of all, thanks for this lib and code.

I don't know why, but when I include this line in the gradle build config:

```
compile 'jp.co.recruit_mp:android-HeaderFooterGridView:0.2'
```

... the app gets a new permission requirement for:

```
read phone status and identity
```

Do you have any idea why this happens? I have checked the code and manifests of the library and I don't see any permission required. However I can reliably repeat this, every time I include the line in the gradle the app get's the new permission requirement, when I remove the line the permission goes away... I use now the java file directly in my project and it's fine that way (no new permission required), I just have no idea why this happens.

Tested on Android 4.4.4. and 5.x, same behavior.

![screenshot-2015-01-26_23 05 32 190](https://cloud.githubusercontent.com/assets/1464667/5909498/6dbe29f4-a5b1-11e4-8121-e41d0de3cf52.png)

**Comment by yhirano** [comment]
Sorry I just have no idea why this happens too...

Workaround:
If you copy a following file to your project, you can use HeaderFooterGridView.
HeaderFooterGridView project is one class only.
https://github.com/recruit-mp/android-HeaderFooterGridView/blob/master/library/src/main/java/jp/co/recruit_mp/android/widget/HeaderFooterGridView.java

**Comment by yhirano** [comment]
I found cause of this issue. I forget that set minSdkVersion and targetSdkVersion to HeaderFooterGridView library.
http://stackoverflow.com/questions/1747178/android-permissions-phone-calls-read-phone-state-and-identity

I'll fix this issue before long.
Thanks your report!

**Comment by zoltanf** [comment]
Thank you, looking forward for the fix on maven.

Best regards,
Zoltan

**Comment by yhirano** [comment]
I fixed your report issue and released v0.2.1.
You'll find the new version library (v0.2.1) after a few hours later from Maven central repository.

Thanks!
